backend/app/models.py

```diff
@@ -286,6 +287,7 @@ class Meta:
             "created",
             "updated",
             "_links",
+            "checksum",
```

Comment:
`_links` field, being a special property, should be the last entry in the generated response. 
Moving it below `git` and above `created` should be fine.